index.js

```diff
@@ -53,14 +51,25 @@ const sendResponse = ( req, settings, fn ) => {
 		}
 
 		let { body, headers, statusCode } = response;
+		const contentType = req.xhr
+			? req.xhr.getResponseHeader( 'Content-Type' )
+			: headers[ 'content-type' ];
```

Comment:
You can always look just at the `headers` parameter, can't you? Is `req.xhr.getResponseHeader` expected to return something different?

Reply:
done in a048023ab8632cf1e737ca29e5d3205b3d4d85a9